src/FakeItEasy/Core/TypeCatalogue.cs

```diff
@@ -168,7 +168,7 @@ private static void WarnFailedToGetTypes(Assembly assembly, Exception ex)
 
         private static void WarnFailedToGetSomeTypes(Assembly assembly, ReflectionTypeLoadException ex)
         {
-            var writer = new DefaultOutputWriter(Console.Write);
+            var writer = new DefaultOutputWriter(Console.Write, null);

```

Comment:
At this point we can't use `ArgumentValueFormatter` because we haven't loaded the custom formatters yet. But it's safe to pass null because we know that the writer won't be used to format argument values.